src/llmcompressor/transformers/finetune/model_args.py

```diff
@@ -83,3 +83,91 @@ class ModelArguments:
             "repositories you trust and in which you have read the code"
         },
     )
+
+
+@dataclass
+class OneshotModelArguments:
```

Comment:
Why do we need to introduce additional model args as opposed to using the model args that we already have defined?

Seems redundant

Reply:
Good question. We got rid of training args. Additional args here are from training args, ex. save_compressed.